src/animation/Transition.mjs

```diff
@@ -197,6 +204,9 @@ export default class Transition extends EventEmitter {
     }
 
 }
+Transition.getGetterProxy = null;
```

Comment:
What do we need those proxy methods for?

Reply:
Well actually we don't need them, i've committed a change